src/inet/UDPEndPointImplSockets.cpp

```diff
@@ -281,6 +281,9 @@ CHIP_ERROR UDPEndPointImplSockets::SendMsgImpl(const IPPacketInfo * aPktInfo, Sy
     // Ensure the destination address type is compatible with the endpoint address type.
     VerifyOrReturnError(mAddrType == aPktInfo->DestAddress.Type(), CHIP_ERROR_INVALID_ARGUMENT);
 
+    // Ensure packet buffer is not null
+    VerifyOrReturnError(!msg.IsNull(), CHIP_ERROR_NO_MEMORY);
```

Comment:
Could you please move this to the very beginning of the checks? We should not validate `GetSocket` outcome if the message is not going to be used.

Reply:
You got it!